src/main/java/com/powsybl/openloadflow/network/DiscreteVoltageControl.java

```diff
@@ -17,15 +17,18 @@
 public class DiscreteVoltageControl {
 
     public enum Mode {
-        VOLTAGE,
+        VOLTAGE_TRANSFORMER,
+        VOLTAGE_SHUNT,
```

Comment:
What was the motivation to share the same object control with transformer one?
This look to me 2 different things.
The implementation looks now very confusing to me as we have 2 modes and 2 controller lists and that both have to be consistent.
What about renaming existing `DiscreteVoltageControl` to `TransformerVoltageControl` and adding a new `ShuntCompensatorVoltageControl` ?

Reply:
Okay, no problem to do that.

Reply:
Done.

Reply:
@flo-dup I think that we can simplify the code now by removing the notion of "discrete" for phase controls.